takeshape/penny
# Clarify env files

* Adds a `prepare-env` script
* Updates README
* Removes / reorgs `.env` files

### Test Plan (Steps to test):

1. Pull this branch
2. Copy your `.env.local` file somewhere temporary
3. Run `npm run prepare-env` — you should be prompted as to whether you want to overwrite files. Answer yes.
1. Review all the overwritten files, see if the instructions make sense.
1. Discard the changes and restore your `.env.local` file.
